components/cache/cache_pools.rst

```diff
@@ -0,0 +1,248 @@
+.. index::
+   single: Cache Pool
+   single: APC Cache, APCu Cache
+   single: Doctrine Cache
+   single: Redis Cache
+
+Cache Pools
+===========
+
+Cache Pools are the logical repositories of cache items. They perform all the
+common operations on items, such as saving them or looking for them. Cache pools
+are independent from the actual cache implementation. Therefore, applications
+can keep using the same cache pool even if the underlying cache mechanism
+changes from a file system based cache to a Redis or database based cache.
+
+Creating Cache Pools
+--------------------
+
+Cache Pools are created through the **cache adapters**, which are classes that
+implement the :class:`Psr\\Cache\\CacheItemPoolInterface` interface. This
```

Comment:
true for symfony derivated adapters (AdapterInterface extends CacheItemPoolInterface)